src/components/Table/Table.mdx

```diff
@@ -49,3 +49,7 @@ This type includes all the configuration for the columns
 ### Column Chooser
 
 <Canvas of={TableStories.ColumnChooser} />
+
+### Sorting
+
```

Comment:
missing documentation on how to use it, multiple vs single, buttons explaining

Reply:
check latest commit for documentation